### PR TITLE
Move CRA tests to DinD, make app connector tests required

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

- Make integration tests required
- Migrate compass runtime agent test to DinD

**Related issue(s)**

### CODE REVIEW

**Review by m00g3n — APPROVED** [review]

**Review by m00g3n — APPROVED** [review]
LGTM

File: prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        prow.k8s.io/pubsub.runID: "pre-main-kyma-integration-k3d-runtime-agent"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-gc-project-env: "true"
+        preset-kyma-guard-bot-github-token: "true"
+        preset-kyma-integration-app-connector-tests-runtime-agent: "true"
+        preset-kyma-integration-central-app-connectivity-enabled: "true"
+        preset-kyma-integration-compass-dev: "true"
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
```
Comment (Ressetkk) [inline prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-tests.yaml:19]: Please use only the required presets for your job

File: prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      spec:
+        containers:
+          - image: "eu.gcr.io/kyma-project/test-infra/kyma-integration:v20230119-993f0759"
+            command:
+              - tests/components/application-connector/scripts/run-tests.sh
+            resources:
+              requests:
+                memory: 1Gi
+              limits:
+                memory: 1Gi
```
Comment (Ressetkk) [inline prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-tests.yaml:47]: Adjust also CPU requests and limits if possible.

**Review by Ressetkk — CHANGES_REQUESTED** [review]

File: prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-tests.yaml (hunk 2 above)
Comment (VOID404) [reply prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-tests.yaml:34]: Is there a sane limit that you'd suggest?

**Review by Ressetkk — APPROVED** [review]

**Comment by VOID404** [comment]
Holding until https://github.com/kyma-project/kyma/pull/17121 passes

**Pull request merged**